src/test/java/io/vertx/openapi/contract/OpenAPIVersionTest.java

```diff
@@ -30,16 +32,19 @@
 import org.junit.jupiter.params.provider.MethodSource;
 
 import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
 import java.util.function.Consumer;
 import java.util.function.Function;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 import static com.google.common.truth.Truth.assertThat;
-import static io.vertx.openapi.ResourceHelper.TEST_RESOURCE_PATH;
-import static io.vertx.openapi.impl.Utils.EMPTY_JSON_OBJECT;
+import static io.vertx.openapi.ResourceHelper.*;
```

Comment:
I'm personally not a friend of star imports, I'd appreciate if we can avoid them.

Reply:
Intellij automatically changing these to * imports. I'll have to update my settings 😄